server/models.py

```
@@ -99,4 +107,48 @@ def __init__(self, sender_id, receiver_id, message):
+            "image_link": image_link,
+            "upvotes": 0,
+            "downvotes": 0,
+            "created_at": datetime.utcnow(),
+            "updated_at": datetime.utcnow()
+        }
+        return posts_collection.insert_one(post)
+
+    @staticmethod
+    def update_post(post_id, description=None, tags=None, image_link=None):
+        updates = {"updated_at": datetime.utcnow()}
+        if description: updates["description"] = description
```

Ruff

Multiple statements on one line (colon)

(E701)